City-of-Helsinki/drupal-helfi-sote
# UHF-2457: filter for units with sv as provided language

How to set up for testing & front-end development:

- Set up the `drupal-helfi-sote` site using this branch
- Import some TPR units: `drush mim tpr_unit --limit=500`
- Go to the Drupal UI and publish some (or all) of the TPR units
- Create a landing page and add a Unit search paragraph on it
- Add some (or all) of the published TPR units in the search and publish the node
- Unit search should now have a checkbox with label "Show units that provide service in Swedish"
- Selecting that checkbox and submitting the search form should filter out units that don't have sv as provided language
